src/govuk/components/notification-banner/_index.scss

```diff
@@ -0,0 +1,87 @@
+@include govuk-exports("govuk/component/notification-banner") {
+  /* stylelint-disable */
+
+  .govuk-notification-banner {
+    @include govuk-font($size: 19);
+    border: $govuk-border-width solid $govuk-brand-colour;
+
+    &:focus {
+      outline: 4px solid $govuk-focus-colour;
+    }
+  }
+
+  .govuk-notification-banner--success {
+    border-color: govuk-colour("green");
+  }
+
+  .govuk-notification-banner--error {
+    border-color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__header {
+    padding: 0 govuk-spacing(3) govuk-spacing(1);
+    color: govuk-colour("white");
+    background-color: $govuk-brand-colour;
+
+
+    @include govuk-media-query($from: tablet) {
+      padding: (govuk-spacing(2) - $govuk-border-width) govuk-spacing(4) govuk-spacing(2);
+    }
+  }
+
+  .govuk-notification-banner--success .govuk-notification-banner__header {
```

Comment:
I think this'd be one place where it makes sense to group the styles for each banner and take advantage of Sass' nesting, for example:

```scss
.govuk-notification-banner--success {
  border-color: govuk-colour("green");

  .govuk-notification-banner__header {
    background-color: govuk-colour("green");
  }
}
```

Reply:
It does make me wonder whether the consistency of links vs the header makes sense though… 😬

Reply:
Sure, I wasn't sure if we had a general rule against nesting 👍 Can you explain what you mean by the consistency of links vs header?